python/cudf_polars/cudf_polars/dsl/expressions/string.py

```diff
@@ -409,6 +410,9 @@ def do_evaluate(
             return Column(
                 plc.strings.replace.replace_multiple(column.obj, target.obj, repl.obj)
             )
+        elif self.name is StringFunction.Name.Reverse:
+            (column,) = columns
+            return Column(plc.strings.reverse.reverse(column.obj))
```

Comment:
Just FYI for future PRs, when contructing `Column` could you add `Column(..., dtype=self.dtype)`